Makefile

```diff
@@ -111,8 +111,8 @@ lint-config: golangci-lint ## Verify golangci-lint linter configuration
 
 .PHONY: yamllint
 yamllint:
-	@files=$$(find testdata -name '*.yaml' ! -path 'testdata/*/dist/*'); \
-    	docker run --rm $$(tty -s && echo "-it" || echo) -v $(PWD):/data cytopia/yamllint:latest $$files -d "{extends: relaxed, rules: {line-length: {max: 120}}}" --no-warnings
+	@files=$$(find testdata -name '*.yaml' ! -path 'testdata/*/dist/*' -o -path 'testdata/*/dist/chart/*.yaml' ! -path 'testdata/*/dist/chart/install.yaml'); \
+	docker run --rm $$(tty -s && echo "-it" || echo) -v $(PWD):/data cytopia/yamllint:latest $$files -d "{extends: relaxed, rules: {line-length: {max: 120}}}" --no-warnings
```

Comment:
@sarthaksarthak9 

I think the ONLY change that we need here to acomplish the goal is:
- Instead of EXCLUDE  all under testdata/dist/* dir we need to exclude ONLY the install.yaml

That means probably replace `'testdata/*/dist/*'` with `! -path 'testdata/*/dist/install.yaml'`